src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './CommentInfo.scss';
+import { Comment } from '../types/Comment';
+
+type Props = {
+  // eslint-disable-next-line react/require-default-props
+  comment?: Comment;
+};
+
+export const CommentInfo: React.FC<Props> = (props) => {
+  const { comment } = props;
+
+  return (
+    <div className="comment">
+      {comment?.body}
```

Comment:
```suggestion
      {comment.body}
```
The component `CommentInfo` is rendered in your app only if THERE IS actually a comment. So props will be always present and there is no need to write `?` sign here